bn_s_mp_to_decimal_fast.c

```diff
@@ -0,0 +1,245 @@
+#include "tommath_private.h"
+#include <string.h>
+#ifdef BN_S_MP_TO_DECIMAL_FAST_C
+/* LibTomMath, multiple-precision integer library -- Tom St Denis */
+/* SPDX-License-Identifier: Unlicense */
+
+/* store a bignum as a decimal ASCII string */
+mp_err s_mp_to_decimal_fast_rec(const mp_int *number, mp_int *nL, mp_int *shiftL, mp_int *mL, int precalc_array_index,
```

Comment:
oh I just saw that this should be static

Reply:
Both `s_mp_to_decimal_fast` and `s_mp_to_decimal_fast_rec` I assume?

Reply:
Oh, looks like not.

Reply:
Done in d2678b0.